packages/playwright/example.js

```diff
@@ -1,12 +1,17 @@
-const AxeBuilder = require('@axe-core/playwright').default;
+const { AxeBuilder } = require('@axe-core/playwright');
 const playwright = require('playwright');
```

Comment:
playwright is outdated (I know this was already here) we should be using playwright-core: 

```suggestion
const playwright = require('playwright-core');
```